packages/components/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "@prenda/components",
   "version": "0.0.1",
   "dependencies": {
+    "@mui/base": "^5.0.0-beta.23",
+    "@mui/material": "^5.14.17",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
These should be peer-deps, not regular deps. 

This `package.json` is what will be published. If we publish this, then consumers have two copies of React. One that they install and import from, and one that we install and import from. No bueno.

For direction, see the [`package.json` of `@mui/material-next`](https://github.com/mui/material-ui/blob/master/packages/mui-material-next/package.json). Ours will look similar.